ilc/server/registry/Registry.js

```diff
@@ -93,4 +101,30 @@ module.exports = class Registry {
 
         return res.data;
     };
+
+    #filterConfig = (config, filter) => {
+        if (!filter || !Object.keys(filter).length) {
+            return config;
+        }
+
+        const clonedConfig = clone(config);
+        const { domain } = filter;
+
+        if (domain) {
```

Comment:
Pls inverse condition here to decrease nesting level.

Reply:
I did it consciously since property `filter` can contain other filters in the future